### PR TITLE
Feat/project mapping

### PR DESCRIPTION
**Pull request by akshay-codemonk**
Add:
- Project mapping model and screen
- Model unit tests
mappings

Fix:
- Background card height for all screens with pagination
- Add media queries for content layout
- workspace_schedule app test case
- Date/time picker widget position in schedule screen
fix_date_time_widget

### CODE REVIEW

File: apps/xero_workspace/forms.py [hunk 1]
```diff
@@ -38,6 +38,20 @@ class EmployeeMappingForm(forms.Form):
...
+    Form for getting project mapping key and value
+    """
+    project_name = forms.CharField(max_length=64, label='', help_text=None,
+                                   widget=forms.TextInput(attrs={'placeholder': ' '}))
+    tracking_category_name = forms.CharField(max_length=64, label='', help_text=None,
+                                             widget=forms.TextInput(attrs={'placeholder': ' '}))
+    tracking_category_option = forms.CharField(max_length=64, label='', help_text=None,
+                                               widget=forms.TextInput(attrs={'placeholder': ' '}))
+    bulk_upload_file = forms.FileField(label='', help_text=None,
+                                       widget=forms.FileInput(attrs={'accept': '.xlsx'}))
```
Comment (snarayanank2) [inline apps/xero_workspace/forms.py]: accept .csv files as well - can be done as a separate task.

File: apps/xero_workspace/static/css/xero_workspace/schedule.css [hunk 2]
```diff
@@ -129,4 +130,10 @@ label {
 
 p {
   margin-right: 5%;
+}
+
+.bootstrap-datetimepicker-widget {
+    top: -130px !important;
```
Comment (snarayanank2) [inline apps/xero_workspace/static/css/xero_workspace/schedule.css]: why was this necessary?

File: apps/xero_workspace/views.py [hunk 3]
```diff
@@ -71,7 +71,8 @@ class DestinationView(View):
     def get(self, request, workspace_id):
         form = XeroCredentialsForm()
         is_connected = XeroCredential.objects.filter(workspace__id=workspace_id).exists()
```
Comment (snarayanank2) [inline apps/xero_workspace/views.py]: naming convention - don't use is_<verb> form. just use <verb>. e.g. "connected"

File: apps/xero_workspace/forms.py (hunk 1 above)
Comment (akshay-codemonk) [reply apps/xero_workspace/forms.py]: Will update this in next MR

File: apps/xero_workspace/static/css/xero_workspace/schedule.css (hunk 2 above)
Comment (akshay-codemonk) [reply apps/xero_workspace/static/css/xero_workspace/schedule.css]: To fix position of date-time picker
error_date_time_widget
fix_date_time_widget

File: apps/xero_workspace/views.py (hunk 3 above)
Comment (akshay-codemonk) [reply apps/xero_workspace/views.py]: Fixed in commit b3dc029

File: apps/xero_workspace/models.py [hunk 4]
```diff
@@ -185,6 +186,6 @@ def create_workspace_(instance, created, **kwargs):
                                            schedule_type=Schedule.MINUTES,
                                            repeats=0,
                                            minutes=5,
-                                           next_run=datetime.datetime.now()
+                                           next_run=datetime.datetime.now(tz=pytz.UTC)
```
Comment (snarayanank2) [inline apps/xero_workspace/models.py]: can we just use utcnow?
